apps/src/acemode/annotationList.js

```diff
@@ -12,6 +13,22 @@ function updateGutter() {
     return;
   }
   aceSession.setAnnotations(annotations);
+
+  if (dropletEditor) {
+    // TODO: connect to missing hover event to show text
+    // TODO: differentiate between error and warning (type is always warning)
```

Comment:
I always encourage folks to attach names to TODO's (or create pivotal items), so that they don't get lost in the ether.